File: assertion-roulette/java-junit/isc_stmt_handle_impl_ESTest.java
@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class isc_stmt_handle_impl_ESTest extends isc_stmt_handle_impl_ESTest_scaffolding {
  @Test(timeout = 4000)
  public void test00()  throws Throwable  {
      isc_stmt_handle_impl isc_stmt_handle_impl0 = new isc_stmt_handle_impl();
      isc_stmt_handle_impl0.ensureCapacity(1010);
      isc_stmt_handle_impl0.ensureCapacity(0);
      assertFalse(isc_stmt_handle_impl0.isAllRowsFetched());
      assertEquals(0, isc_stmt_handle_impl0.size());
      assertFalse(isc_stmt_handle_impl0.isSingletonResult());
  }
}

File: smelly/assertion-roulette/CSharpXUnit/CSharpXUnit.Tests/Tests/isc_stmt_handle_impl_ESTest.cs
[Collection("EvoRunner")]
public class isc_stmt_handle_impl_ESTest : isc_stmt_handle_impl_ESTest_scaffolding
{
    [Fact(Timeout = 4000)]
    public void test00()
    {
        var isc_stmt_handle_impl0 = new isc_stmt_handle_impl();
        isc_stmt_handle_impl0.ensureCapacity(1010);
        isc_stmt_handle_impl0.ensureCapacity(0);
        Assert.False(isc_stmt_handle_impl0.isAllRowsFetched());
        Assert.Equal(0, isc_stmt_handle_impl0.size());
        Assert.False(isc_stmt_handle_impl0.isSingletonResult());
    }
}
